sdk/spring/azure-spring-boot-test-selenium-common/src/main/java/com/azure/test/aad/common/SeleniumITHelper.java

```diff
@@ -48,4 +71,24 @@ public void destroy() {
         driver.quit();
         app.close();
     }
+
+    private void setPathExecutableRecursively(File file) {
+        if (!file.exists()) {
+            logger.warn("Path " + file + " does not exist!");
+            return;
+        }
+
+        if (file.isDirectory()) {
+            File[] files = file.listFiles();
+            if (null != files && files.length > 0) {
```

Comment:
Can we delete this check?
Refs: https://stackoverflow.com/questions/24845281/how-does-a-for-each-loop-guard-against-an-empty-list